SoftLayer/CLI/account/invoices.py

```diff
@@ -0,0 +1,44 @@
+"""Invoice listing"""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+from SoftLayer.managers.account import AccountManager as AccountManager
+from SoftLayer import utils
+
+
+@click.command()
+@click.option('--limit', default=50, show_default=True,
+              help="How many invoices to get back.")
+@click.option('--closed', is_flag=True, default=False, show_default=True,
+              help="Include invoices with a CLOSED status.")
+@click.option('--all', 'get_all', is_flag=True, default=False, show_default=True,
+              help="Return ALL invoices. There may be a lot of these.")
+@environment.pass_env
+def cli(env, limit, closed=False, get_all=False):
+    """Invoices and all that mess"""
+
+    manager = AccountManager(env.client)
+    invoices = manager.get_invoices(limit, closed, get_all)
+
+    table = formatting.Table([
+        "Id", "Created", "Type", "Status", "Starting Balance", "Ending Balance", "Invoice Amount", "Items"
+    ])
+    table.align['Starting Balance'] = 'l'
+    table.align['Ending Balance'] = 'l'
+    table.align['Invoice Amount'] = 'l'
+    table.align['Items'] = 'l'
+    for invoice in invoices:
+        table.add_row([
+            invoice.get('id'),
```

Comment:
if value limit is 1 (e.g `slcli account invoices --limit 1`), an exception  will throw  as below :
```
invoice.get('id'),
AttributeError: 'str' object has no attribute 'get'
```